### ISSUE TITLE
Tag release?

### ISSUE THREAD

**Issue by mAAdhaTTah** [wp-sync-db/wp-sync-db#28]
If you could tag a release, it would be easier for me to lock wp-sync-db to a particular version with composer. Thanks!

**Comment by slang800** [comment]
Tag in what way? I've already pushed the git tags (https://github.com/wp-sync-db/wp-sync-db/releases)... does composer rely upon something else?

**Comment by mAAdhaTTah** [comment]
No, it relies on those tags, but the tags were added before you added the plugin to Packagist (and thus don't have a composer.json file). You would just need to tag it again now that the package is up there, and Packagist will read the new git tag.

**Comment by peterjohnjoseph** [comment]
+1 on this.

**Comment by lsmith77** [comment]
ping?

**Comment by tremby** [comment]
Yes please.

**Comment by valentinocossar** [comment]
+1 on this.
